### ISSUE TITLE
Interval.equals breaks when not valid

### ISSUE THREAD

**Issue by dodo** [moment/luxon#262]
```javascript
a = Interval.invalid("foo")
b = Interval.invalid("bar")

a.equals(b)
// => TypeError: Cannot read property 'equals' of undefined
a.start === undefined
// => true
```
Well that escalated quickly.

i guess it's probably just one line missing ;)
```javascript
if (!this.isValid) return false;
```

**Comment by icambron** [comment]
Yup. Would take a PR for that

**Comment by icambron** [comment]
Fixed in 9fc9dc4. Will release soon
